libkbfs/keybase_service_base.go

```diff
@@ -197,7 +199,21 @@ func filterRevokedKeys(
 		var info revokedKeyInfo
 		if key.Base.Revocation != nil {
 			info.Time = key.Base.Revocation.Time
-			info.MerkleRoot = key.Base.Revocation.PrevMerkleRootSigned
+			res, err := k.userClient.FindNextMerkleRootAfterRevoke(ctx,
```

Comment:
Is this server trust or does it verify the revoke signature is included in the merkle root? If former, do you think we can prefer `PrevMerkleRootSigned` when it's greater than zero, since it's from a client device?